Makefile

```diff
@@ -1,6 +1,11 @@
 GIT_HOST = sigs.k8s.io
 PWD := $(shell pwd)
 BASE_DIR := $(shell basename $(PWD))
+# customize kubebuilder path
+KUBEBUILDER_PATH ?= /usr/local
```

Comment:
we've already have download scripts for all dependency binaries. Is it possible to reuse it instead of managing a new download method here?